ci/kokoro/macos/build-bazel.sh

```diff
@@ -52,6 +52,15 @@ if [[ -n "${BAZEL_CONFIG}" ]]; then
     bazel_args+=("--config" "${BAZEL_CONFIG}")
 fi
 
+echo
+echo "================================================================"
+for repeat in 1 2 3; do
+  echo "Fetch bazel dependencies at $(date) [${repeat}/3]."
+  if "${BAZEL_BIN}" fetch -- //google/cloud/...:all; then
```

Comment:
Is this any different than `bazel sync`?

Also, FWIW, the `:all` shouldn't be needed there. I believe that's equivalent to just `//google/cloud/...`. No change necessary, JFYI.

Reply:
`bazel sync` downloads all dependencies, regardless of whether they are needed, since `googleapis` includes targets for Java and Go I try to avoid those.

TIL about `:all`.